### PR TITLE
[fix #6] [mozmeao/infra#1104] Clean up oregon-b prod yaml.

### LINKED ISSUE #6: clean up yaml for oregon-b prod
related to mozmeao/infra#1104

### PR DESCRIPTION
**Pull request by jpetto**
- rename svc.yaml to web-svc.yaml

### CODE REVIEW

File: oregon-b/basket-prod/clock-deploy.yaml [hunk 1]
```diff
@@ -48,381 +38,256 @@ spec:
...
-            secretKeyRef:
-              key: celery-always-eager
-              name: basket-prod-v50-env
+          value: "False"
         - name: CELERY_NUM_WORKERS
-          valueFrom:
-            secretKeyRef:
-              key: celery-num-workers
-              name: basket-prod-v50-env
+          value: 5
```
Comment (jgmize) [inline oregon-b/basket-prod/clock-deploy.yaml]:
```suggestion
          value: "5"
```

File: oregon-b/basket-prod/clock-deploy.yaml [hunk 2]
```diff
@@ -48,381 +38,256 @@ spec:
...
         - name: CELERY_REDIS_MAX_CONNECTIONS
-          valueFrom:
-            secretKeyRef:
-              key: celery-redis-max-connections
-              name: basket-prod-v50-env
+          value: 5
```
Comment (jgmize) [inline oregon-b/basket-prod/clock-deploy.yaml]:
```suggestion
          value: "5"
```

File: oregon-b/basket-prod/clock-deploy.yaml [hunk 3]
```diff
@@ -48,381 +38,256 @@ spec:
...
             secretKeyRef:
               key: email-host-user
-              name: basket-prod-v50-env
+              name: basket-prod-secrets
         - name: EMAIL_PORT
-          valueFrom:
-            secretKeyRef:
-              key: email-port
-              name: basket-prod-v50-env
+          value: 587
```
Comment (jgmize) [inline oregon-b/basket-prod/clock-deploy.yaml]:
```suggestion
          value: "587"
```

File: oregon-b/basket-prod/clock-deploy.yaml [hunk 4]
```diff
@@ -48,381 +38,256 @@ spec:
...
             secretKeyRef:
               key: port
-              name: basket-prod-v50-env
+              name: basket-prod-secrets
         - name: REDIS_CACHE_DB
-          valueFrom:
-            secretKeyRef:
-              key: redis-cache-db
-              name: basket-prod-v50-env
+          value: 5
```
Comment (jgmize) [inline oregon-b/basket-prod/clock-deploy.yaml]:
```suggestion
          value: "5"
```

File: oregon-b/basket-prod/clock-deploy.yaml [hunk 5]
```diff
@@ -48,381 +38,256 @@ spec:
...
         - name: REDIS_CELERY_DB
-          valueFrom:
-            secretKeyRef:
-              key: redis-celery-db
-              name: basket-prod-v50-env
+          value: 4
```
Comment (jgmize) [inline oregon-b/basket-prod/clock-deploy.yaml]:
```suggestion
          value: "4"
```

File: oregon-b/basket-prod/clock-deploy.yaml [hunk 6]
```diff
@@ -48,381 +38,256 @@ spec:
...
-          valueFrom:
-            secretKeyRef:
-              key: workflow-release-created-at
-              name: basket-prod-v50-env
-        - name: WORKFLOW_RELEASE_SUMMARY
-          valueFrom:
-            secretKeyRef:
-              key: workflow-release-summary
-              name: basket-prod-v50-env
+          value: 10
```
Comment (jgmize) [inline oregon-b/basket-prod/clock-deploy.yaml]:
```suggestion
          value: "10"
```

File: oregon-b/basket-prod/clock-deploy.yaml [hunk 7]
```diff
@@ -48,381 +38,256 @@ spec:
...
         - name: WSGI_NUM_WORKERS
-          valueFrom:
-            secretKeyRef:
-              key: wsgi-num-workers
-              name: basket-prod-v50-env
+          value: 4
```
Comment (jgmize) [inline oregon-b/basket-prod/clock-deploy.yaml]:
```suggestion
          value: "4"
```

File: oregon-b/basket-prod/clock-deploy.yaml [hunk 8]
```diff
@@ -48,381 +38,256 @@ spec:
...
         - name: conflicts
-          valueFrom:
-            secretKeyRef:
-              key: conflicts
-              name: basket-prod-v50-env
-        image: 127.0.0.1:5555/basket-prod:v50
+          value: 1
```
Comment (jgmize) [inline oregon-b/basket-prod/clock-deploy.yaml]:
```suggestion
          value: "1"
```

File: oregon-b/basket-prod/clock-deploy.yaml [hunk 9]
```diff
@@ -48,381 +38,256 @@ spec:
...
+        image: mozmeao/basket:4d0668d
```
Comment (jgmize) [inline oregon-b/basket-prod/clock-deploy.yaml]:
```suggestion
        image: mozmeao/basket:4d0668d4e5cb304e68b04bcc0d2b278d27b591e6
```

File: oregon-b/basket-prod/donateworker-deploy.yaml [hunk 10]
```diff
@@ -48,381 +38,256 @@ spec:
...
-            secretKeyRef:
-              key: celery-always-eager
-              name: basket-prod-v50-env
+          value: "False"
         - name: CELERY_NUM_WORKERS
-          valueFrom:
-            secretKeyRef:
-              key: celery-num-workers
-              name: basket-prod-v50-env
+          value: 5
```
Comment (jgmize) [inline oregon-b/basket-prod/donateworker-deploy.yaml]:
```suggestion
          value: "5"
```

File: oregon-b/basket-prod/donateworker-deploy.yaml [hunk 11]
```diff
@@ -48,381 +38,256 @@ spec:
...
         - name: CELERY_REDIS_MAX_CONNECTIONS
-          valueFrom:
-            secretKeyRef:
-              key: celery-redis-max-connections
-              name: basket-prod-v50-env
+          value: 5
```
Comment (jgmize) [inline oregon-b/basket-prod/donateworker-deploy.yaml]:
```suggestion
          value: "5"
```

File: oregon-b/basket-prod/donateworker-deploy.yaml [hunk 12]
```diff
@@ -48,381 +38,256 @@ spec:
...
             secretKeyRef:
               key: email-host-user
-              name: basket-prod-v50-env
+              name: basket-prod-secrets
         - name: EMAIL_PORT
-          valueFrom:
-            secretKeyRef:
-              key: email-port
-              name: basket-prod-v50-env
+          value: 587
```
Comment (jgmize) [inline oregon-b/basket-prod/donateworker-deploy.yaml]:
```suggestion
          value: "587"
```

File: oregon-b/basket-prod/donateworker-deploy.yaml [hunk 13]
```diff
@@ -48,381 +38,256 @@ spec:
...
             secretKeyRef:
               key: port
-              name: basket-prod-v50-env
+              name: basket-prod-secrets
         - name: REDIS_CACHE_DB
-          valueFrom:
-            secretKeyRef:
-              key: redis-cache-db
-              name: basket-prod-v50-env
+          value: 5
```
Comment (jgmize) [inline oregon-b/basket-prod/donateworker-deploy.yaml]:
```suggestion
          value: "5"
```

File: oregon-b/basket-prod/donateworker-deploy.yaml [hunk 14]
```diff
@@ -48,381 +38,256 @@ spec:
...
         - name: REDIS_CELERY_DB
-          valueFrom:
-            secretKeyRef:
-              key: redis-celery-db
-              name: basket-prod-v50-env
+          value: 4
```
Comment (jgmize) [inline oregon-b/basket-prod/donateworker-deploy.yaml]:
```suggestion
          value: "4"
```

File: oregon-b/basket-prod/donateworker-deploy.yaml [hunk 15]
```diff
@@ -48,381 +38,256 @@ spec:
...
-          valueFrom:
-            secretKeyRef:
-              key: workflow-release-created-at
-              name: basket-prod-v50-env
-        - name: WORKFLOW_RELEASE_SUMMARY
-          valueFrom:
-            secretKeyRef:
-              key: workflow-release-summary
-              name: basket-prod-v50-env
+          value: 10
```
Comment (jgmize) [inline oregon-b/basket-prod/donateworker-deploy.yaml]:
```suggestion
          value: "10"
```

File: oregon-b/basket-prod/donateworker-deploy.yaml [hunk 16]
```diff
@@ -48,381 +38,256 @@ spec:
...
         - name: WSGI_NUM_WORKERS
-          valueFrom:
-            secretKeyRef:
-              key: wsgi-num-workers
-              name: basket-prod-v50-env
+          value: 4
```
Comment (jgmize) [inline oregon-b/basket-prod/donateworker-deploy.yaml]:
```suggestion
          value: "4"
```

File: oregon-b/basket-prod/donateworker-deploy.yaml [hunk 17]
```diff
@@ -48,381 +38,256 @@ spec:
...
         - name: conflicts
-          valueFrom:
-            secretKeyRef:
-              key: conflicts
-              name: basket-prod-v50-env
-        image: 127.0.0.1:5555/basket-prod:v50
+          value: 1
```
Comment (jgmize) [inline oregon-b/basket-prod/donateworker-deploy.yaml]:
```suggestion
          value: "1"
```

File: oregon-b/basket-prod/donateworker-deploy.yaml [hunk 18]
```diff
@@ -48,381 +38,256 @@ spec:
...
+        image: mozmeao/basket:4d0668d
```
Comment (jgmize) [inline oregon-b/basket-prod/donateworker-deploy.yaml]:
```suggestion
        image: mozmeao/basket:4d0668d4e5cb304e68b04bcc0d2b278d27b591e6
```

File: oregon-b/basket-prod/fxaeventsworker-deploy.yaml [hunk 19]
```diff
@@ -48,381 +37,256 @@ spec:
...
-            secretKeyRef:
-              key: celery-always-eager
-              name: basket-prod-v50-env
+          value: "False"
         - name: CELERY_NUM_WORKERS
-          valueFrom:
-            secretKeyRef:
-              key: celery-num-workers
-              name: basket-prod-v50-env
+          value: 5
```
Comment (jgmize) [inline oregon-b/basket-prod/fxaeventsworker-deploy.yaml]:
```suggestion
          value: "5"
```

File: oregon-b/basket-prod/fxaeventsworker-deploy.yaml [hunk 20]
```diff
@@ -48,381 +37,256 @@ spec:
...
         - name: CELERY_REDIS_MAX_CONNECTIONS
-          valueFrom:
-            secretKeyRef:
-              key: celery-redis-max-connections
-              name: basket-prod-v50-env
+          value: 5
```
Comment (jgmize) [inline oregon-b/basket-prod/fxaeventsworker-deploy.yaml]:
```suggestion
          value: "5"
```

File: oregon-b/basket-prod/fxaeventsworker-deploy.yaml [hunk 21]
```diff
@@ -48,381 +37,256 @@ spec:
...
             secretKeyRef:
               key: email-host-user
-              name: basket-prod-v50-env
+              name: basket-prod-secrets
         - name: EMAIL_PORT
-          valueFrom:
-            secretKeyRef:
-              key: email-port
-              name: basket-prod-v50-env
+          value: 587
```
Comment (jgmize) [inline oregon-b/basket-prod/fxaeventsworker-deploy.yaml]:
```suggestion
          value: "587"
```

File: oregon-b/basket-prod/fxaeventsworker-deploy.yaml [hunk 22]
```diff
@@ -48,381 +37,256 @@ spec:
...
             secretKeyRef:
               key: port
-              name: basket-prod-v50-env
+              name: basket-prod-secrets
         - name: REDIS_CACHE_DB
-          valueFrom:
-            secretKeyRef:
-              key: redis-cache-db
-              name: basket-prod-v50-env
+          value: 5
```
Comment (jgmize) [inline oregon-b/basket-prod/fxaeventsworker-deploy.yaml]:
```suggestion
          value: "5"
```

File: oregon-b/basket-prod/fxaeventsworker-deploy.yaml [hunk 23]
```diff
@@ -48,381 +37,256 @@ spec:
...
         - name: REDIS_CELERY_DB
-          valueFrom:
-            secretKeyRef:
-              key: redis-celery-db
-              name: basket-prod-v50-env
+          value: 4
```
Comment (jgmize) [inline oregon-b/basket-prod/fxaeventsworker-deploy.yaml]:
```suggestion
          value: "4"
```

File: oregon-b/basket-prod/fxaeventsworker-deploy.yaml [hunk 24]
```diff
@@ -48,381 +37,256 @@ spec:
...
-          valueFrom:
-            secretKeyRef:
-              key: workflow-release-created-at
-              name: basket-prod-v50-env
-        - name: WORKFLOW_RELEASE_SUMMARY
-          valueFrom:
-            secretKeyRef:
-              key: workflow-release-summary
-              name: basket-prod-v50-env
+          value: 10
```
Comment (jgmize) [inline oregon-b/basket-prod/fxaeventsworker-deploy.yaml]:
```suggestion
          value: "10"
```

File: oregon-b/basket-prod/fxaeventsworker-deploy.yaml [hunk 25]
```diff
@@ -48,381 +37,256 @@ spec:
...
         - name: WSGI_NUM_WORKERS
-          valueFrom:
-            secretKeyRef:
-              key: wsgi-num-workers
-              name: basket-prod-v50-env
+          value: 4
```
Comment (jgmize) [inline oregon-b/basket-prod/fxaeventsworker-deploy.yaml]:
```suggestion
          value: "4"
```

File: oregon-b/basket-prod/fxaeventsworker-deploy.yaml [hunk 26]
```diff
@@ -48,381 +37,256 @@ spec:
...
         - name: conflicts
-          valueFrom:
-            secretKeyRef:
-              key: conflicts
-              name: basket-prod-v50-env
-        image: 127.0.0.1:5555/basket-prod:v50
+          value: 1
```
Comment (jgmize) [inline oregon-b/basket-prod/fxaeventsworker-deploy.yaml]:
```suggestion
          value: "1"
```

File: oregon-b/basket-prod/fxaeventsworker-deploy.yaml [hunk 27]
```diff
@@ -48,381 +37,256 @@ spec:
...
+        image: mozmeao/basket:4d0668d
```
Comment (jgmize) [inline oregon-b/basket-prod/fxaeventsworker-deploy.yaml]:
```suggestion
        image: mozmeao/basket:4d0668d4e5cb304e68b04bcc0d2b278d27b591e6
```

File: oregon-b/basket-prod/fxaworker-deploy.yaml [hunk 28]
```diff
@@ -48,381 +37,256 @@ spec:
...
-            secretKeyRef:
-              key: celery-always-eager
-              name: basket-prod-v50-env
+          value: "False"
         - name: CELERY_NUM_WORKERS
-          valueFrom:
-            secretKeyRef:
-              key: celery-num-workers
-              name: basket-prod-v50-env
+          value: 5
```
Comment (jgmize) [inline oregon-b/basket-prod/fxaworker-deploy.yaml]:
```suggestion
          value: "5"
```

File: oregon-b/basket-prod/fxaworker-deploy.yaml [hunk 29]
```diff
@@ -48,381 +37,256 @@ spec:
...
         - name: CELERY_REDIS_MAX_CONNECTIONS
-          valueFrom:
-            secretKeyRef:
-              key: celery-redis-max-connections
-              name: basket-prod-v50-env
+          value: 5
```
Comment (jgmize) [inline oregon-b/basket-prod/fxaworker-deploy.yaml]:
```suggestion
          value: "5"
```

File: oregon-b/basket-prod/fxaworker-deploy.yaml [hunk 30]
```diff
@@ -48,381 +37,256 @@ spec:
...
             secretKeyRef:
               key: email-host-user
-              name: basket-prod-v50-env
+              name: basket-prod-secrets
         - name: EMAIL_PORT
-          valueFrom:
-            secretKeyRef:
-              key: email-port
-              name: basket-prod-v50-env
+          value: 587
```
Comment (jgmize) [inline oregon-b/basket-prod/fxaworker-deploy.yaml]:
```suggestion
          value: "587"
```

File: oregon-b/basket-prod/fxaworker-deploy.yaml [hunk 31]
```diff
@@ -48,381 +37,256 @@ spec:
...
             secretKeyRef:
               key: port
-              name: basket-prod-v50-env
+              name: basket-prod-secrets
         - name: REDIS_CACHE_DB
-          valueFrom:
-            secretKeyRef:
-              key: redis-cache-db
-              name: basket-prod-v50-env
+          value: 5
```
Comment (jgmize) [inline oregon-b/basket-prod/fxaworker-deploy.yaml]:
```suggestion
          value: "5"
```

File: oregon-b/basket-prod/fxaworker-deploy.yaml [hunk 32]
```diff
@@ -48,381 +37,256 @@ spec:
...
         - name: REDIS_CELERY_DB
-          valueFrom:
-            secretKeyRef:
-              key: redis-celery-db
-              name: basket-prod-v50-env
+          value: 4
```
Comment (jgmize) [inline oregon-b/basket-prod/fxaworker-deploy.yaml]:
```suggestion
          value: "4"
```

File: oregon-b/basket-prod/fxaworker-deploy.yaml [hunk 33]
```diff
@@ -48,381 +37,256 @@ spec:
...
-          valueFrom:
-            secretKeyRef:
-              key: workflow-release-created-at
-              name: basket-prod-v50-env
-        - name: WORKFLOW_RELEASE_SUMMARY
-          valueFrom:
-            secretKeyRef:
-              key: workflow-release-summary
-              name: basket-prod-v50-env
+          value: 10
```
Comment (jgmize) [inline oregon-b/basket-prod/fxaworker-deploy.yaml]:
```suggestion
          value: "10"
```

File: oregon-b/basket-prod/ns.yaml [hunk 34]
```diff
@@ -1,13 +1,7 @@
...
 kind: Namespace
 metadata:
-  creationTimestamp: "2018-01-24T22:03:44Z"
-  labels:
-    heritage: deis
   name: basket-prod
-  resourceVersion: "1120259"
-  selfLink: /api/v1/namespaces/basket-prod
-  uid: 71ae6221-0152-11e8-8c8b-0656fabf84c0
 spec:
```
Comment (jgmize) [inline oregon-b/basket-prod/ns.yaml]: Please remove the spec & status sections

File: oregon-b/basket-prod/web-deploy.yaml [hunk 35]
```diff
@@ -48,381 +37,256 @@ spec:
...
             secretKeyRef:
               key: email-host-user
-              name: basket-prod-v50-env
+              name: basket-prod-secrets
         - name: EMAIL_PORT
-          valueFrom:
-            secretKeyRef:
-              key: email-port
-              name: basket-prod-v50-env
+          value: 587
```
Comment (jgmize) [inline oregon-b/basket-prod/web-deploy.yaml]:
```suggestion
          value: "587"
```

File: oregon-b/basket-prod/web-deploy.yaml [hunk 36]
```diff
@@ -48,381 +37,256 @@ spec:
...
-            secretKeyRef:
-              key: celery-always-eager
-              name: basket-prod-v50-env
+          value: "False"
         - name: CELERY_NUM_WORKERS
-          valueFrom:
-            secretKeyRef:
-              key: celery-num-workers
-              name: basket-prod-v50-env
+          value: 5
```
Comment (jgmize) [inline oregon-b/basket-prod/web-deploy.yaml]:
```suggestion
          value: "5"
```

File: oregon-b/basket-prod/web-deploy.yaml [hunk 37]
```diff
@@ -48,381 +37,256 @@ spec:
...
         - name: CELERY_REDIS_MAX_CONNECTIONS
-          valueFrom:
-            secretKeyRef:
-              key: celery-redis-max-connections
-              name: basket-prod-v50-env
+          value: 5
```
Comment (jgmize) [inline oregon-b/basket-prod/web-deploy.yaml]:
```suggestion
          value: "5"
```

File: oregon-b/basket-prod/web-deploy.yaml [hunk 38]
```diff
@@ -48,381 +37,256 @@ spec:
...
             secretKeyRef:
               key: port
-              name: basket-prod-v50-env
+              name: basket-prod-secrets
         - name: REDIS_CACHE_DB
-          valueFrom:
-            secretKeyRef:
-              key: redis-cache-db
-              name: basket-prod-v50-env
+          value: 5
```
Comment (jgmize) [inline oregon-b/basket-prod/web-deploy.yaml]:
```suggestion
          value: "5"
```

File: oregon-b/basket-prod/web-deploy.yaml [hunk 39]
```diff
@@ -48,381 +37,256 @@ spec:
...
         - name: REDIS_CELERY_DB
-          valueFrom:
-            secretKeyRef:
-              key: redis-celery-db
-              name: basket-prod-v50-env
+          value: 4
```
Comment (jgmize) [inline oregon-b/basket-prod/web-deploy.yaml]:
```suggestion
          value: "4"
```

File: oregon-b/basket-prod/web-deploy.yaml [hunk 40]
```diff
@@ -48,381 +37,256 @@ spec:
...
-          valueFrom:
-            secretKeyRef:
-              key: workflow-release-created-at
-              name: basket-prod-v50-env
-        - name: WORKFLOW_RELEASE_SUMMARY
-          valueFrom:
-            secretKeyRef:
-              key: workflow-release-summary
-              name: basket-prod-v50-env
+          value: 10
```
Comment (jgmize) [inline oregon-b/basket-prod/web-deploy.yaml]:
```suggestion
          value: "10"
```

File: oregon-b/basket-prod/web-deploy.yaml [hunk 41]
```diff
@@ -48,381 +37,256 @@ spec:
...
         - name: WSGI_NUM_WORKERS
-          valueFrom:
-            secretKeyRef:
-              key: wsgi-num-workers
-              name: basket-prod-v50-env
+          value: 4
```
Comment (jgmize) [inline oregon-b/basket-prod/web-deploy.yaml]:
```suggestion
          value: "4"
```

File: oregon-b/basket-prod/web-deploy.yaml [hunk 42]
```diff
@@ -48,381 +37,256 @@ spec:
...
         - name: conflicts
-          valueFrom:
-            secretKeyRef:
-              key: conflicts
-              name: basket-prod-v50-env
-        image: 127.0.0.1:5555/basket-prod:v50
+          value: 1
```
Comment (jgmize) [inline oregon-b/basket-prod/web-deploy.yaml]:
```suggestion
          value: "1"
```

File: oregon-b/basket-prod/web-deploy.yaml [hunk 43]
```diff
@@ -48,381 +37,256 @@ spec:
...
+        image: mozmeao/basket:4d0668d
```
Comment (jgmize) [inline oregon-b/basket-prod/web-deploy.yaml]:
```suggestion
        image: mozmeao/basket:4d0668d4e5cb304e68b04bcc0d2b278d27b591e6
```

File: oregon-b/basket-prod/web-svc.yaml [hunk 44]
```diff
@@ -0,0 +1,23 @@
...
+    nodePort: 32162
+    port: 80
+    protocol: TCP
+    targetPort: 8000
+  selector:
+    app: basket-prod
+    type: web
+  sessionAffinity: None
+  type: NodePort
+status:
```
Comment (jgmize) [inline oregon-b/basket-prod/web-svc.yaml]: Please remove the status section

File: oregon-b/basket-prod/worker-deploy.yaml [hunk 45]
```diff
@@ -48,381 +37,256 @@ spec:
...
-            secretKeyRef:
-              key: celery-always-eager
-              name: basket-prod-v50-env
+          value: "False"
         - name: CELERY_NUM_WORKERS
-          valueFrom:
-            secretKeyRef:
-              key: celery-num-workers
-              name: basket-prod-v50-env
+          value: 5
```
Comment (jgmize) [inline oregon-b/basket-prod/worker-deploy.yaml]:
```suggestion
          value: "5"
```

File: oregon-b/basket-prod/worker-deploy.yaml [hunk 46]
```diff
@@ -48,381 +37,256 @@ spec:
...
         - name: CELERY_REDIS_MAX_CONNECTIONS
-          valueFrom:
-            secretKeyRef:
-              key: celery-redis-max-connections
-              name: basket-prod-v50-env
+          value: 5
```
Comment (jgmize) [inline oregon-b/basket-prod/worker-deploy.yaml]:
```suggestion
          value: "5"
```

File: oregon-b/basket-prod/worker-deploy.yaml [hunk 47]
```diff
@@ -48,381 +37,256 @@ spec:
...
             secretKeyRef:
               key: email-host-user
-              name: basket-prod-v50-env
+              name: basket-prod-secrets
         - name: EMAIL_PORT
-          valueFrom:
-            secretKeyRef:
-              key: email-port
-              name: basket-prod-v50-env
+          value: 587
```
Comment (jgmize) [inline oregon-b/basket-prod/worker-deploy.yaml]:
```suggestion
          value: "587"
```

File: oregon-b/basket-prod/worker-deploy.yaml [hunk 48]
```diff
@@ -48,381 +37,256 @@ spec:
...
             secretKeyRef:
               key: port
-              name: basket-prod-v50-env
+              name: basket-prod-secrets
         - name: REDIS_CACHE_DB
-          valueFrom:
-            secretKeyRef:
-              key: redis-cache-db
-              name: basket-prod-v50-env
+          value: 5
```
Comment (jgmize) [inline oregon-b/basket-prod/worker-deploy.yaml]:
```suggestion
          value: "5"
```

File: oregon-b/basket-prod/worker-deploy.yaml [hunk 49]
```diff
@@ -48,381 +37,256 @@ spec:
...
         - name: REDIS_CELERY_DB
-          valueFrom:
-            secretKeyRef:
-              key: redis-celery-db
-              name: basket-prod-v50-env
+          value: 4
```
Comment (jgmize) [inline oregon-b/basket-prod/worker-deploy.yaml]:
```suggestion
          value: "4"
```

File: oregon-b/basket-prod/worker-deploy.yaml [hunk 50]
```diff
@@ -48,381 +37,256 @@ spec:
...
-          valueFrom:
-            secretKeyRef:
-              key: workflow-release-created-at
-              name: basket-prod-v50-env
-        - name: WORKFLOW_RELEASE_SUMMARY
-          valueFrom:
-            secretKeyRef:
-              key: workflow-release-summary
-              name: basket-prod-v50-env
+          value: 10
```
Comment (jgmize) [inline oregon-b/basket-prod/worker-deploy.yaml]:
```suggestion
          value: "10"
```

File: oregon-b/basket-prod/worker-deploy.yaml [hunk 51]
```diff
@@ -48,381 +37,256 @@ spec:
...
         - name: WSGI_NUM_WORKERS
-          valueFrom:
-            secretKeyRef:
-              key: wsgi-num-workers
-              name: basket-prod-v50-env
+          value: 4
```
Comment (jgmize) [inline oregon-b/basket-prod/worker-deploy.yaml]:
```suggestion
          value: "4"
```

File: oregon-b/basket-prod/worker-deploy.yaml [hunk 52]
```diff
@@ -48,381 +37,256 @@ spec:
...
         - name: conflicts
-          valueFrom:
-            secretKeyRef:
-              key: conflicts
-              name: basket-prod-v50-env
-        image: 127.0.0.1:5555/basket-prod:v50
+          value: 1
```
Comment (jgmize) [inline oregon-b/basket-prod/worker-deploy.yaml]:
```suggestion
          value: "1"
```

File: oregon-b/basket-prod/worker-deploy.yaml [hunk 53]
```diff
@@ -48,381 +37,256 @@ spec:
...
+        image: mozmeao/basket:4d0668d
```
Comment (jgmize) [inline oregon-b/basket-prod/worker-deploy.yaml]:
```suggestion
        image: mozmeao/basket:4d0668d4e5cb304e68b04bcc0d2b278d27b591e6
```

**Review by jgmize — CHANGES_REQUESTED** [review]

**Comment by jpetto** [comment]
@jgmize okay, this should be ready for final review. apologies for missing those few original review comments.

**Review by jgmize — APPROVED** [review]
Thanks @jpetto

**Pull request merged**